src/sum/sum.ts

```diff
@@ -0,0 +1,2 @@
+// Placeholder
```

Comment:
Could we rename this file to `src/index.ts`? So that is clear that is going to be the entry point of our lib.

Reply:
The file is more like "proof of concept": typescript is present and works. Once actual code is written in the `lib` will use `index.js`.